xml/fencing-1.2.rng

```diff
@@ -18,10 +18,7 @@
   <define name="element-level">
     <element name="fencing-level">
       <attribute name="id"><data type="ID"/></attribute>
-      <choice>
-	<attribute name="target"><text/></attribute>
-	<attribute name="target-pattern"><text/></attribute>
-      </choice>
+      <attribute name="target"><text/></attribute>
       <attribute name="index"><data type="positiveInteger"/></attribute>
```

Comment:
I shouldn't have done this.  I don't really recall why I thought it was a good idea.
However we've done up and downstream releases since then that include it, so removing it now is probably a bad idea.